server/views/partials/layout.njk

```diff
@@ -41,7 +41,13 @@
 
 {% block header %}
   {% include "./header.njk" %}
-  {% include "./locationBar.njk" %}
+
+  {% if prisonGroup %}
```

Comment:
I thought about using `not prisonGroup`, but decided to use the same logic as `server/views/partials/analytics/layout.njk`